internal/uvm/update_uvm.go

```diff
@@ -0,0 +1,56 @@
+package uvm
+
+import (
+	"context"
+
+	hcsschema "github.com/Microsoft/hcsshim/internal/schema2"
+	specs "github.com/opencontainers/runtime-spec/specs-go"
+)
+
+func (uvm *UtilityVM) UpdateConstraints(ctx context.Context, data interface{}, annotations map[string]string) error {
+	var memoryLimitInBytes *uint64
+	var processorLimits *hcsschema.ProcessorLimits
+
+	switch resources := data.(type) {
+	case *specs.WindowsResources:
```

Comment:
I don't like overloading the resource values on the OCI spec to also be used for UVM resources. We don't do this at UVM creation time, so I don't think we should do so here either. I think we should just look up the CPU/memory annotations that we use at UVM creation time, and use them here as well.

Reply:
That's a fair point. That'll be a pretty big change here though. Would you be okay with me removing the support for cpu update for a pod in this PR and adding it back using annotations in a later PR?

Reply:
Yes, that's fine